apps/marketplace/src/app/item/[songId]/page.tsx

```diff
@@ -29,63 +30,69 @@ const SingleSong: FunctionComponent<SingleSongProps> = ({ params }) => {
   }
 
   return (
-    <Container maxWidth="md" sx={ { flexGrow: 1, mt: [0, 0, 5] } }>
-      <Stack
-        alignItems={ ["center", "center", "start"] }
-        direction={ ["column", "column", "row"] }
-      >
-        <Box mb={ [2, 2, 0] } mr={ [0, 0, 5] } width={ [240, 240, 400] }>
-          <SongCard
-            coverArtUrl={ songData?.coverArtUrl }
-            imageDimensions={ 480 }
-            isPlayable={ true }
-            price={ mockSongs[0].price }
-            // eslint-disable-next-line @typescript-eslint/no-empty-function
-            onCardClick={ () => {} }
-            // eslint-disable-next-line @typescript-eslint/no-empty-function
-            onPriceClick={ () => {} }
-            // eslint-disable-next-line @typescript-eslint/no-empty-function
-            onSubtitleClick={ () => {} }
-          />
-        </Box>
+    <Container>
+      <Container maxWidth="md" sx={ { flexGrow: 1, mt: [0, 0, 5] } }>
```

Comment:
A container within a container seems stranger than the other wrapper components, Stacks on Stacks, should we migrate the single song card/description/metadata into its own component? I can refactor this as part of the buy function/UI I'm working on if you agree.